javascript/code-challenges/challenges-07.test.js

```diff
@@ -0,0 +1,359 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called sortStarWarsCharacters that sorts the characters in the starWarsPeople array by height from tallest to shortest.
+------------------------------------------------------------------------------------------------ */
+
+let starWarsPeople = [
+  {
+    "name": "C-3PO",
+    "height": "167",
+    "eye_color": "yellow"
+  },
+  {
+    "name": "Luke Skywalker",
+    "height": "172",
+    "eye_color": "blue"
+  },
+  {
+    "name": "R2-D2",
+    "height": "96",
+    "eye_color": "red"
+  }
+];
+
+const sortStarWarsCharacters = (starWarsArr) => {
+  return starWarsPeople.sort((a, b) => b.height - a.height);
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named removeThree that takes an index and an array. The function should removes three items in the array starting with the value at the index. 
+------------------------------------------------------------------------------------------------ */
+
+const removeThree = (idx, arr) => {
+  return arr.splice(0, 3);
```

Comment:
Two reasons this isn't passing
1. You need to run **.splice** on the arr and THEN return the array.
2. You want the first argument in your **arr.splice** to be **idx** not 0.